Sources/MSLXCTest.example/ExampleTests.swift

```diff
@@ -0,0 +1,41 @@
+import MSLCombine
+import MSLXCTest
+import XCTest
+
+class TestService {
+    @PublisherConvertible
+    var testValue = false
+    lazy var testValuePublisher = self.$testValue
+
+    func updateTestValue(_ newValue: Bool) {
+        testValue = newValue
+    }
+}
+
+class TestObservableObject: ObservableObject {
+    @Published
+    var testValue = 0
+
+    func updateTestValue(_ newValue: Int) {
+        testValue = newValue
+    }
+}
+
+final class ExampleTest: XCTestCase {
```

Comment:
I'm a little confused as to the purpose of this file/test. Is this intended to test `PublisherConvertible`?